src/test/java/burp/RegexTest.java

```diff
@@ -109,13 +109,12 @@ public void testFalsePositives() {
             
 	    if (foundMatches >= 1) { 
                 matchCount++;
-            } else {
-                System.out.println("Unable to find match for: " + rule.getPattern());
+                System.out.println("Found false positive for: " + rule.getPattern());
             }
         }
         
         System.out.println(String.format("Found %d matches out of %d", matchCount, matchRules.size()));
-        assertEquals(matchCount, 0);
+        assertEquals(0, matchCount);
```

Comment:
No, it just improves the error message if it fails. The first parameter should be the expected value. If this fails, it will now emit a message like "expected: 0, actual: 1".

Reply:
Nice. Thanks!